skiko/src/commonMain/kotlin/org/jetbrains/skiko/Events.kt

```diff
@@ -237,5 +237,8 @@ data class SkikoPointer(
     val pressure: Double = 1.0,
 )
 
-internal val Iterable<SkikoPointer>.centroidX get() = asSequence().map { it.x }.average()
-internal val Iterable<SkikoPointer>.centroidY get() = asSequence().map { it.y }.average()
\ No newline at end of file
+@DelicateSkikoApi
+val Iterable<SkikoPointer>.centroidX get() = asSequence().map { it.x }.average()
+
+@DelicateSkikoApi
```

Comment:
Why do you need to add a new `SkikoInternalApi` but marking it with different `DelicateSkikoApi` annotation?

Reply:
Yep, I am wrong here.